ui/raidboss/data/03-hw/alliance/weeping_city.js

```diff
@@ -0,0 +1,329 @@
+      condition: function(data) {
+        return data.role == 'tank' || data.role == 'healer';
+      },
+      infoText: {
+        en: 'Get orbs',
+      },
+    },
+    {
+      id: 'Weeping City Acceleration Bomb',
+      netRegex: NetRegexes.gainsEffect({ effectId: '430' }),
+      condition: Conditions.targetIsYou(),
+      delaySeconds: function(data, matches) {
+        return parseFloat(matches.duration) - 3;
+      },
+      response: Responses.stopEverything(),
+    },
+    {
+      id: 'Weeping City Assimilation',
+      netRegex: NetRegexes.startsUsing({ id: '1802', source: 'Ozmashade', capture: false }),
+      response: Responses.lookAway(),
+    },
+    {
+      // Each party gets a stack marker, so this is the best we can do.
+      id: 'Weeping City Meteor Stack',
+      netRegex: NetRegexes.headMarker({ id: '003E', capture: false }),
+      condition: function(data) {
+        return data.ozmaStarted;
+      },
+      suppressSeconds: 5,
+      response: Responses.stack(),
+    },
+    {
+      // Coif Change is always followed up shortly by Haircut.
+      // There's no castbar or indicator except that she grows a scythe on one side.
+      // It's not a very obvious visual cue unless the player knows to look for it.
+      id: 'Weeping City Coif Change Left',
+      netRegex: NetRegexes.ability({ id: '180A', source: 'Calofisteri', capture: false }),
+      response: Responses.goRight(),
+    },
+    {
+      id: 'Weeping City Coif Change Right',
+      netRegex: NetRegexes.ability({ id: '180E', source: 'Calofisteri', capture: false }),
+      response: Responses.goLeft(),
+    },
+    {
+      // 4899 is the base ID for bulb hair. 4900 is axe hair.
+      // Bulbs do a circle AoE surrounding them, while axes are a donut.
+      id: 'Weeping City Living Lock Axes',
+      netRegex: NetRegexes.addedCombatantFull({ npcNameId: ['4899', '4900'], capture: false }),
+      suppressSeconds: 5,
+      infoText: {
+        en: 'Close to axes, far from bulbs',
+      },
+    },
+    {
+      id: 'Weeping City Living Lock Scythes',
+      netRegex: NetRegexes.addedCombatantFull({ npcNameId: '4898', capture: false }),
+      suppressSeconds: 5,
+      alertText: {
+        en: 'Avoid scythe line AoEs',
+      },
+    },
+    {
+      // These adds are the purple circles waiting to grab people and Garrotte them.
+      id: 'Weeping City Entanglement',
+      netRegex: NetRegexes.addedCombatantFull({ npcNameId: '4904', capture: false }),
+      suppressSeconds: 5,
+      infoText: {
+        en: 'Avoid purple circles',
+      },
+    },
+    {
+      // If by some chance someone actually did stand in the purple circles, break them out.
+      // The actual ability here is an Unknown ability, but it begins slightly before Garrotte.
+      id: 'Weeping City Garrotte',
+      netRegex: NetRegexes.ability({ id: '181D', source: 'Entanglement', capture: false }),
+      response: Responses.killExtraAdd(),
```

Comment:
This needs a suppressSeconds on it, because it calls for every person in it.

Reply:
Seriously? That's really weird. I could have sworn that it was a single self-targeted cast. Oh well, as you say it's an easy fix.

Reply:
```
[20:02:59.077] 03:40005680:Added new combatant Entanglement.  Job: N/A Level: 60 Max HP: 27040 Max MP: 10000 Pos: (-305.0634,-27.36672,-436.1) (49040000005736).
[20:03:07.565] 16:40005680:Entanglement:181D:Unknown_181D:1001AF20:Dahlia Frandrezka:21:440000:E:29C0000:1B:181D8000:0:0:0:0:0:0:0:0:0:0:26965:26967:10000:10000:0:1000:-303.0169:-26.16636:-436.1:-1.569665:27040:27040:8840:10000:0:1000:-305.0634:-27.36672:-436.1:-4.792213E-05:0000F777
[20:03:07.565] 16:40005680:Entanglement:181D:Unknown_181D:1068652A:Crystalyze Menot:21:440000:E:29C0000:1B:181D8000:0:0:0:0:0:0:0:0:0:0:23920:23920:10000:10000:0:1000:-309.8345:-23.3006:-436.1:1.89932:27040:27040:8840:10000:0:1000:-305.0634:-27.36672:-436.1:-4.792213E-05:0000F777
[20:03:07.565] 16:40005680:Entanglement:181D:Unknown_181D:106F402B:Myelia Jadeline:21:440000:E:29C0000:1B:181D8000:0:0:0:0:0:0:0:0:0:0:24136:24136:1736:10000:0:1000:-306.2066:-21.45212:-436.1092:2.929516:27040:27040:8840:10000:0:1000:-305.0634:-27.36672:-436.1:-4.792213E-05:0000F777
[20:03:07.565] 16:40005680:Entanglement:181D:Unknown_181D:106C0D5E:Yue Kasai:21:440000:E:29C0000:1B:181D8000:0:0:0:0:0:0:0:0:0:0:21796:21796:10000:10000:0:1000:-298.6343:-25.16223:-436.1:-2.981577:27040:27040:8840:10000:0:1000:-305.0634:-27.36672:-436.1:-4.792213E-05:0000F777
[20:03:07.565] 16:40005680:Entanglement:181D:Unknown_181D:1060E77F:Evelyn Lace:21:440000:E:29C0000:1B:181D8000:0:0:0:0:0:0:0:0:0:0:22332:22332:10000:10000:0:1000:-298.6954:-25.43683:-436.1:-2.983974:27040:27040:8840:10000:0:1000:-305.0634:-27.36672:-436.1:-4.792213E-05:0000F777
```